assets/js/blocks/cart-checkout/cart/test/block.js

```diff
@@ -40,13 +40,7 @@ describe( 'Testing cart', () => {
 			screen.getByText( /Proceed to Checkout/i )
 		).toBeInTheDocument();
 
-		/**
-		 * @todo Investigate extra POST request on initial cart render.
-		 *
-		 * We have an unfixed bug in our test in which the full cart triggers a POST
-		 * request to `wc/store/cart/update-item` causing the fetch to be called twice.
-		 */
-		expect( fetchMock ).toHaveBeenCalledTimes( 2 );
```

Comment:
💃 Finally fixed!

Reply:
Should probably also close this
https://github.com/woocommerce/woocommerce-gutenberg-products-block/issues/2813

Reply:
Ah, yes. Thanks for pointing to the issue. I assigned it to me and linked it with this PR.